src/engine/controls/cuecontrol.cpp

```diff
@@ -680,11 +674,9 @@ void CueControl::hotcueSet(HotcueControl* pControl, double v) {
 void CueControl::hotcueGoto(HotcueControl* pControl, double v) {
     if (!v)
         return;
-
     QMutexLocker lock(&m_mutex);
-    if (!m_pLoadedTrack) {
+    if (!m_pLoadedTrack)
         return;
-    }
```

Comment:
This should stay as is, we always use braces.

Reply:
This is a case where I would disagree, even though I agree in general. This is just a guarding clause, so there should _never_ be anything other than the return in there. The current coding style in the wiki also allows this.

Reply:
> The current coding style in the wiki also allows this. 

... for historical reasons. It states clearly that the version with braces is better. This change is a regression, because it reduces the Better version to just OK.

Reply:
Have you had a look at the file and compared them? Originally, these statements were supposed to be one-liners, but the formatter adjusted that. If you look at it as a whole, each line introduced by these simple repetitive statements is only going to create more noise, and that includes an extra line of braces.

I still don't see any case where you would add anything to this if statement, so omitting the braces allows you to take mental shortcuts and ignore them to some degree. It differentiates them from the if statements that are actually interesting.